pkg/ddc/alluxio/shutdown.go

```diff
@@ -138,6 +144,34 @@ func (e *AlluxioEngine) cleanupCache() (err error) {
 	return fmt.Errorf("the remaining cached is not cleaned up, check again")
 }
 
+func (e *AlluxioEngine) releasePorts() (err error) {
+	//TODO(xuzhihao): Move this function to utils
+	var valueConfigMapName = e.name + "-" + e.runtimeType + "-values"
+
+	allocator, err := portallocator.GetRuntimePortAllocator()
```

Comment:
I think the initialization of portallocator should be moved to initialization of the runtime controller. When did it do in Kubernetes?